rasa/core/channels/mattermost.py

```diff
@@ -87,26 +130,49 @@ def blueprint(self, on_new_message):
         async def webhook(request: Request):
             output = request.json
             if output:
```

Comment:
I would change the if clause to avoid nested code. E.g.
```
if not output:
    return response.text("")

if "trigger_word" in output:
    ....
```

Reply:
Done